components/widget/standings/widget_standings_swiss.lua

```diff
@@ -0,0 +1,140 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Widget/Standings/Swiss
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Class = require('Module:Class')
+local Lua = require('Module:Lua')
+local Table = require('Module:Table')
+
+local WidgetUtil = Lua.import('Module:Widget/Util')
+local Widget = Lua.import('Module:Widget')
+local HtmlWidgets = Lua.import('Module:Widget/Html/All')
+local DataTable = Lua.import('Module:Widget/Basic/DataTable')
+local MatchOverview = Lua.import('Module:Widget/Standings/MatchOverview')
+
+local OpponentLibraries = require('Module:OpponentLibraries')
+local OpponentDisplay = OpponentLibraries.OpponentDisplay
+
+---@class StandingsSwissWidget: Widget
+---@operator call(table): StandingsSwissWidget
+
+local StandingsSwissWidget = Class.new(Widget)
+
+---@return Widget?
+function StandingsSwissWidget:render()
+	if not self.props.standings then
+		return
+	end
+
+	---@type StandingsModel
+	local standings = self.props.standings
+	local activeRounds = (Array.maxBy(
+		Array.filter(standings.rounds, function(round) return round.started end),
+		function (round) return round.round end
+	) or {round = 0}).round
+	local hasFutureRounds = not standings.rounds[#standings.rounds].started
+
+	return DataTable{
+		wrapperClasses = {'standings-ffa', 'toggle-area', 'toggle-area-' .. activeRounds},
+		classes = {'wikitable-bordered', 'wikitable-striped'},
+		attributes = {['data-toggle-area'] = activeRounds},
+		children = WidgetUtil.collect(
+			-- Outer header
+			HtmlWidgets.Tr{children = HtmlWidgets.Th{
+				attributes = {
+					colspan = 100,
+				},
+				children = {
+					HtmlWidgets.Div{
+						css = {['position'] = 'relative'},
+						children = {
+							HtmlWidgets.Span{
+								children = standings.title
+							},
+						},
+					},
+				},
+			}},
+			-- Column Header
+			HtmlWidgets.Tr{children = WidgetUtil.collect(
+				HtmlWidgets.Th{children = '#'},
+				HtmlWidgets.Th{children = 'Participant'},
+				HtmlWidgets.Th{children = 'Matches'},
```

Comment:
fwiw will need options for Games, GameDIfference, evetually buchholz columns

Reply:
Yeah I know, I'll add it when tiebreakers comes around I think. I don't see the need to add the information if it's not part of the resolve